doc/Type/Signature.pod6

```diff
@@ -668,9 +668,9 @@ These are called "slurpy" because they slurp up any remaining arguments
 to a function, like someone slurping up noodles.
 
 =begin code
-$ = :($a, @b);  # exactly two arguments, where the second one must be Positional
-$ = :($a, *@b); # at least one argument, @b slurps up any beyond that
-$ = :(*%h);     # no positional arguments, but any number of named arguments
+:($a, @b);  # exactly two arguments, where the second one must be Positional
```

Comment:
I'm going to assume this works, but in that case it might be better to show these examples in a more real context, or simply eliminate the `;`, which gives the (possibly confusing) impression that this would be a valid statement.

BTW, there are similar assignments further down in the document, you might wan to change that in the same PR.